### ISSUE TITLE
Time travel is broken

### ISSUE THREAD

**Issue by fenhl** [fenhl/lore-seeker#2]
Looks like the time travel fix to get future Custom Standard rotations working broke the feature in other ways. Details to be investigated.

**Comment by taw** [comment]
Oh interesting. Time travel has 4 components:

* card release date - that should be straightforward, just stick proper date in set's json file (or per file if it's a weird set like promo cards). Time travel gets really confused when this date is wrong or missing.
* name resolution - if you use `time:roe` instead of `time:2010-15-31` format, it uses that date, easy to support for your own sets
* ban list - that generally shouldn't apply, but banlist format is super friendly (see `search-engine/lib/ban_list/brawl.rb` for small recent example). It's just list of dates with list of legalities and announcement link (can be `nil`).
* format rotation - if you inherit from `FormatStandard` and override `rotation_schedule`, it should work.

There's no rotation-specific logic anywhere else IIRC.

**Comment by taw** [comment]
I'd recommend writing specs like `search-engine/spec/format_brawl_spec.rb` for your custom rotating format.

**Comment by fenhl** [comment]
Sorry, there's some missing context here. The “fix” I'm referring to is [this change](https://github.com/fenhl/lore-seeker/compare/master...lore-seeker#diff-c7f364b820b7c1eee06bbb46201c56f6R59), intent being that cards with a future release date are not shown by default; you'd have to use time travel if you want to look at future rotations. It works well for that purpose, but I don't exactly remember what this broke.

**Comment by fenhl** [comment]
Found it again: the query `(time:OGW f:Standard) not (time:SOI f:Standard)` [works fine](https://mtg.wtf/card?q=%28time%3AOGW+f%3AStandard%29+not+%28time%3ASOI+f%3AStandard%29) on mtg.wtf but [is broken](https://loreseeker.fenhl.net/card?q=%28time%3AOGW+f%3AStandard%29+not+%28time%3ASOI+f%3AStandard%29) on Lore Seeker.

**Comment by taw** [comment]
Right, the default is to include future cards, so it works in prerelease season. You need explicit `time:now` or similar to override this.

**Comment by fenhl** [comment]
That's not the desired bahavior for Lore Seeker though, hence my (broken) change.
